osgi-dependencies/selenium/pom.xml

```diff
@@ -67,7 +67,7 @@
     <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-htmlunit-driver</artifactId>
-      <version>${selenium.version}</version>
+      <version>2.52.0</version>
```

Comment:
I've seen that selenium htmlunit package was separated from the selenium itself some time ago. And I believe this version has connection with all ignored unit tests?
Is there any way to make those ignored unit tests working? If not, maybe we should just remove them?
Or maybe we should add ignore comment like:
`@Ignore("Not working because of selenium-htmlunit-driver broken compatibility. Consider to remove in the future.")`

Reply:
We are irrespectively  working on fixing or even rewriting failing tests. Probably soon there will be additional pull request for junit changes.